### DISCUSSION TITLE
How to fix flickering exactly?

### DISCUSSION THREAD

**Discussion by joeyhead** [KingSora/OverlayScrollbars#511] [Q&A] [answered]
Hi,

Reading [Bridging initialization flickering](https://github.com/KingSora/OverlayScrollbars#bridging-initialization-flickering) I learned that I'm supposed to do the following to fix flickering:

> To fix this behavior apply the `data-overlayscrollbars-initialize` attribute to the target element (and `html` element if the target element is `body`).

Unfortunately I'm not sure how to do that. Would you please give me a walkthrough?

I'm using OverlayScrollbars 2.1.0 within Wordpress and loading it to the body this way:

HTML:
```
<script src=" https://cdn.jsdelivr.net/npm/overlayscrollbars@2.1.0/browser/overlayscrollbars.browser.es6.min.js "></script>
<link href=" https://cdn.jsdelivr.net/npm/overlayscrollbars@2.1.0/styles/overlayscrollbars.min.css " rel="stylesheet">
```

JS (jQuery):
```
 jQuery(document).ready(function($) {
  var OverlayScrollbars = window.OverlayScrollbarsGlobal.OverlayScrollbars;
  OverlayScrollbars(document.body, { 
    scrollbars: {
      theme: 'os-theme-light',
    },
  });
});
```

Thank you.

**Comment by KingSora** [comment, marked as answer]
Good day @joeyhead 

So in your case because you apply the plugin to the `body` element you have to add the `data-overlayscrollbars-initialize` attribute to your `html` element.
Before the change your html file looks something like this:
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body></body>
</html>
```
and after:
```html
<!DOCTYPE html>
<html data-overlayscrollbars-initialize>
  <head></head>
  <body></body>
</html>
```

In case you add it to a other element like a div you do the same just with the div:
```html
<div data-overlayscrollbars-initialize>
  OverlayScrollbars is applied to this div
</div>
```

**Comment by joeyhead** [comment]
Thank you very much, it worked great. :)

**Reply by KingSora** [reply]
Youre welcome :)
